test/e2e/plain_provisioner_test.go

```diff
@@ -1202,9 +1202,9 @@ var _ = Describe("plain provisioner bundle", func() {
 						Not(BeNil()),
 						WithTransform(func(c *metav1.Condition) string { return c.Type }, Equal(rukpakv1alpha2.TypeInstalled)),
 						WithTransform(func(c *metav1.Condition) metav1.ConditionStatus { return c.Status }, Equal(metav1.ConditionFalse)),
-						WithTransform(func(c *metav1.Condition) string { return c.Reason }, Equal(rukpakv1alpha2.ReasonInstallFailed)),
+						WithTransform(func(c *metav1.Condition) string { return c.Reason }, Equal(rukpakv1alpha2.ReasonErrorGettingReleaseState)),
```

Comment:
Is this happening because we're now doing a dry-run install that is failing?

Reply:
Correct